[2021-06-25 15:17:00,901] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available 2021-06-24T00:00:00+00:00 [queued]>
[2021-06-25 15:17:00,907] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: user_processing.is_api_available 2021-06-24T00:00:00+00:00 [queued]>
[2021-06-25 15:17:00,907] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 15:17:00,907] {taskinstance.py:1068} INFO - Starting attempt 5 of 5
[2021-06-25 15:17:00,907] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-06-25 15:17:00,913] {taskinstance.py:1087} INFO - Executing <Task(HttpSensor): is_api_available> on 2021-06-24T00:00:00+00:00
[2021-06-25 15:17:00,916] {standard_task_runner.py:52} INFO - Started process 31799 to run task
[2021-06-25 15:17:00,921] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'user_processing', 'is_api_available', '2021-06-24T00:00:00+00:00', '--job-id', '10', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/var/folders/9t/2jnn4bv95w9552ldckktgq900000gn/T/tmpy2lj_mrq', '--error-file', '/var/folders/9t/2jnn4bv95w9552ldckktgq900000gn/T/tmpmotl_60o']
[2021-06-25 15:17:00,922] {standard_task_runner.py:77} INFO - Job 10: Subtask is_api_available
[2021-06-25 15:17:00,940] {logging_mixin.py:104} INFO - Running <TaskInstance: user_processing.is_api_available 2021-06-24T00:00:00+00:00 [running]> on host saidamirs-macbook-pro.local
[2021-06-25 15:17:00,954] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=is_api_available
AIRFLOW_CTX_EXECUTION_DATE=2021-06-24T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-06-24T00:00:00+00:00
[2021-06-25 15:17:00,954] {http.py:101} INFO - Poking: api/
[2021-06-25 15:17:00,956] {base.py:69} INFO - Using connection to: id: user_api. Host: https://randomuser.me/, Port: None, Schema: , Login: , Password: None, extra: {}
[2021-06-25 15:17:00,957] {http.py:140} INFO - Sending 'GET' to url: https://randomuser.me/api/
[2021-06-25 15:17:01,027] {local_task_job.py:151} INFO - Task exited with return code Negsignal.SIGABRT
